src/wallet/constants.ts

```diff
@@ -0,0 +1,12 @@
+export const CB_SW_PROXY_BYTECODE =
+  '0x363d3d373d3d363d7f360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc545af43d6000803e6038573d6000fd5b3d6000f3';
+/**
+ * This is the keccak-256 hash of "eip1967.proxy.implementation" subtracted by 1.
+ * A consistent location where proxies store the address of the logic contract they delegate to,
+ * as well as other proxy-specific information.
+ * https://eips.ethereum.org/EIPS/eip-1967
+ */
+export const ERC_1967_PROXY_IMPLEMENTATION_SLOT =
+  '0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc';
+
+export const COINBASE_SMART_WALLET_V1_IMPLEMENTATION = '0x000100abaad02f1cfC8Bbe32bD5a564817339E72';
```

Comment:
Keep the const variable alphabeticall ordered.

Reply:
also you can just say `CB_SW_V1_IMPLEMENTATION_ADDRESS`, so it's similar to `CB_SW_PROXY_BYTECODE`.